### DISCUSSION TITLE
Provide Ruby as a language option for functions

### DISCUSSION THREAD

**Discussion by dan-gamble** [Shopify/function-examples#376] [Feedback: General Improvements]
With the announcement of [Ruvy](https://shopify.engineering/introducing-ruvy) it'd be amazing to see Ruby as a language option for Shopify Functions.

**Comment by resistorsoftware** [comment]
**I for another agree!** The explanation I received as to why that was not possible **before** Ruvy, was due to the overhead of the _runtime_. In my mind, if they can accommodate Javascript, and Ruvy reduces overheads, then the whole world of Functions becomes much nicer for a lot of developers.
